libraries/entities/src/EntitiesAuditLogging.cpp

```diff
@@ -0,0 +1,84 @@
+//
+//  EntitiesAuditLogging.cpp
+//  libraries/entities/src
+//
+//  Created by Kalila L on Feb 5 2021.
+//  Copyright 2021 Vircadia contributors.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+#include "EntitiesAuditLogging.h"
+
+#include <QJsonObject>
+#include <QTimer> 
+
+Q_LOGGING_CATEGORY(entities_audit, "vircadia.entities.audit");
+
+QJsonObject auditLogAddBuffer;
+QJsonObject auditLogEditBuffer;
+QTimer* auditLogProcessorTimer;
+
+void EntitiesAuditLogging::processAuditLogBuffers() {
+    if (!auditLogAddBuffer.isEmpty()) {
+        QJsonObject objectToOutput;
+        objectToOutput.insert("add", auditLogAddBuffer);
+        qCDebug(entities_audit) << objectToOutput;
+        auditLogAddBuffer = QJsonObject();
+    }
+    if (!auditLogEditBuffer.isEmpty()) {
+        QJsonObject objectToOutput;
+        objectToOutput.insert("edit", auditLogEditBuffer);
+        qCDebug(entities_audit) << objectToOutput;
+        auditLogEditBuffer = QJsonObject();
+    }
+}
+
+void EntitiesAuditLogging::startAuditLogProcessor() {
+    auditLogProcessorTimer = new QTimer(this);
```

Comment:
I think this line may be the problem.  does this need to take this as an argument?

Reply:
I removed it but that didn't really change anything, I'm still in the case of it reporting that the timer is indeed running and audit logging is also on, but the processor isn't doing anything...